qkxtm/QKXTM_util.cpp

```diff
@@ -1617,6 +1616,46 @@ int TSM_NdumpHP = 0;
 int TSM_NdumpLP = 0;
 long int TSM_maxiter = 0;
 double TSM_tol = 0;
+int smethod = 1;
```

Comment:
I would suggest to change the default smethod to "smethod = 0", as it's the one we have been using so far.